.github/workflows/build.yml

```diff
@@ -57,8 +57,11 @@ jobs:
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
 
       - if: inputs.msrv == 'MSRV'
-        name: Prepare minimal package versions (MSRV only)
-        run: nox -s set-minimal-package-versions
+        name: Prepare package versions for MSRV
+        run: |
+          sed -i '/pyo3-ffi-check/d' Cargo.toml  # we allow pyo3-ffi-check to use newer packages, as an internal tool
+          git diff Cargo.toml
+          nox -s set-minimal-package-versions
```

Comment:
Can we just run the various commands with `--workspace --exclude pyo3-ffi-check`?